torch/csrc/jit/codegen/cuda/runtime/random_numbers.cu

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
why was this line removed?

Reply:
no reason, it was random whitespace at the top of the file.